playbooks/zookeeper.yml

```diff
@@ -71,3 +136,22 @@
       pause:
         prompt: "Press Enter to Proceed to Next Node. Ctrl + C to Abort"
       when: zookeeper_pause_rolling_deployment|bool
+
+- name: Zookeeper Parallel Provisioning
```

Comment:
There is a change in the order of parallel and serial provisioning here. The reason is, parallel ordering depends on current/actual properties (Let's say `zookeeper_ssl` was off before and we are trying to turn it on in this run), so now dynamic_groups.yml needs to get run with older properties(i.e. `zookeeper_ssl` off in this example), as it is a read operation, and currently setup is running with older/actual property/state.
And then new provisioning should consider the desired property (for e.g, here desired property is `zookeeper_ssl` to be on). So new order is ->
1. Read the Leader/follower using dynamic_groups (refer SSL off to hit the correct port)
2. Do a serial provisioning if deployment_strategy is serial
3. Do parallel provisioning, if deployment_strategy is parallel.
4. Reflect back the temporary arrangement of custom_properties, to the correct (now desired == actual) property set after the zookeeper setup is done, so next plays like kafka_broker should refer to the current zookeeper properties, and not the transient ones.